languages/python/reference/exercise-concepts/robot-simulator.md

```diff
@@ -0,0 +1,62 @@
+# Concepts for robot-simulator
+
+[Example Implementation](https://github.com/exercism/python/blob/master/exercises/robot-simulator/example.py)
+
+
+## General
+
+- contstants: Are not enforced by the runtime, but are used via the convention of `<ALL_CAPS_NAMING>` to signal that these values are expected to remaine unchanged.  Perferably, constants are defined at a module level. The example solution uses the ALL_CAPS convention to define NORTH, SOUTH, EAST, and WEST constants. 
+
+- multiple assignment:  Python allows multiple assignment, assiging the items on the left of `=` _in order_ to the values on the right of `=`.  Equivelent to forming tuples of each side and then "unpacking" them.  This exercise solution uses multiple assignment for the contant values NORTH, EAST, SOUTH, WEST.
```

Comment:
Spelling: "assiging" -> "assigning", "Equivelent" -> "Equivalent". Not sure this is worded the most clearly (tuples are actually being formed on each side, that's what the comma does, but I'm glad to see it called out.

Reply:
Yup.  I think this probably needs more massaging.  Went with the wording below,  but it probably needs futher edits:

> - multiple assignment:  Python allows multiple assignment, assigning the items on the left of `=` _in order_ to the values on the right of `=` by forming tuples and then "unpacking" them.  This exercise solution uses multiple assignment for the constant values NORTH, EAST, SOUTH, WEST.

Reply:
I'm okay with the new wording.